CMakeLists.txt

```diff
@@ -125,6 +126,11 @@ if( BUILD_WEB_SERVER )
 
 endif()
 
+if( BUILD_AUTHZ_TESTS )
```

Comment:
I thought these lines were added as part of #1157. Why would these changes show in the diff?

Reply:
I think that GitHub doesn't always update the file diffs after merging into the target branch. This branch was built off the earlier one let me pull in the changes and see if it shows correctly.